src/components/ContainerSettingsStats.react.js

```diff
@@ -0,0 +1,355 @@
+import _ from 'underscore';
+import React from 'react/addons';
+import shell from 'shell';
+import ContainerUtil from '../utils/ContainerUtil';
+import containerActions from '../actions/ContainerActions';
+import containerStore from '../stores/ContainerStore';
+import metrics from '../utils/MetricsUtil';
+import docker from '../utils/DockerUtil';
+import {DropdownButton, MenuItem} from 'react-bootstrap';
+import Chart from 'chart.js'
+
+var ContainerSettingsStats = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+  getInitialState: function () {
+
+    this.charts = {
+      cpu: null,
+      memory: null,
+      networkIn: null,
+      networkOut: null
+    }
+
+    return {
+      cpu: {
+        used: 0,
+        free: 0,
+      },
+      memory: {
+        used: 0,
+        total: 0,
+        perc: 0
+      },
+      network: {
+        in: 0,
+        out: 0
+      }
+    };
+  },
+  componentDidMount: function() {
+    var self = this;
+    var pre_networks = {};
+
+    docker.stats(this.props.container.Name, function(error, stream){
+      if(error){
+        console.log('Err: %o', error);
+      }else{
+        self.stream = stream;
+        self.initCharts();
+
+        stream.setEncoding('utf8');
+        stream.on('data', json => {
+          let data = JSON.parse(json);
+          if (data.error) {
+            error = data.error;
+            return;
+          }else{
+
+            // CPU
+            var previousCPU = data.precpu_stats.cpu_usage.total_usage;
+				    var previousSystem = data.precpu_stats.system_cpu_usage;
+            var cpuDelta = parseFloat(data.cpu_stats.cpu_usage.total_usage) - parseFloat(previousCPU)
+		        var systemDelta = parseFloat(data.cpu_stats.system_cpu_usage) - parseFloat(previousSystem)
+	          var cpuPercent = (cpuDelta / systemDelta) * parseFloat(data.cpu_stats.cpu_usage.percpu_usage.length) * 100.0;
```

Comment:
If you are using Windows I applied a patch based on https://github.com/docker/docker/blob/eb131c5383db8cac633919f82abad86c99bffbe5/cli/command/container/stats_helpers.go#L190-L204